adoc/chapters/programming_interface.adoc

```diff
@@ -21660,6 +21666,12 @@ All function objects obey {cpp} conversion and promotion rules.
 Each function object is additionally specialized for [code]#void# as a
 _transparent_ function object that deduces its parameter types and return type.
 
+{note}Using these function objects with program-defined types requires those
+types to define the associated operator (e.g., [code]#plus# requires
+[code]#pass:[+]#).
```

Comment:
> e.g., `plus` requires `+`

Is `+` the same as `operator+()`?

Reply:
Agreed.  Probably better to say `operator+` here.

Reply:
Fixed in 8c5e0636041f0b4c2bd924c7e96ed038c1066c29.